backend/portal/api/portal_api.py

```diff
@@ -434,6 +425,72 @@ def get_collection_index():
     return make_response(jsonify(response), 200)
 
 
+def get_user_private_collections(token_info: Optional[dict] = None):
+    user_info = UserInfo(token_info)
+
+    # Get all private collections the user has write access to
+    if user_info.is_none():
+        all_user_private_collections = []
+    elif user_info.is_super_curator():
+        all_user_private_collections = get_business_logic().get_collections(CollectionQueryFilter(is_published=False))
```

Comment:
this is a better var name than in the previous implementation above, before you modularized it 👍

Reply:
TY!